assets/model_monitoring/components/src/model_monitor_metric_publisher/azure_monitor_metric_publisher.py

```diff
@@ -0,0 +1,104 @@
+import datetime
+import json
+import requests
+
+from pyspark.sql import Row
+from typing import List
+
+from shared_utilities.log_utils import log_utils
+
+def publish_metric(
```

Comment:
We should probably only publish metrics from the metric publisher component - the outputter only creates the monitoring metric files right?

Reply:
Yes, the outputter changes are not relevant in PR (simple rename), which I can move out to another one for ease to review.

Reply:
Moved the irrelevant changes out.